Show "manual flush" on Display with Temperature

genkigenki opened the pull request:
Show "manual flush" on Display with Temperature

fiendie commented:
You committed a backup of your userConfig.h
Also, if you're already rebasing, please squash the "Rebase" commit into the previous one.

genkigenki commented:
> You committed a backup of your userConfig.h Also, if you're already rebasing, please squash the "Rebase" commit into the previous one.

Done.

genkigenki commented:
Changed:
- Showing brewtime (hint from @LoQue90 - thanks)
- Moved duplicated code to show brew time into function
- Analogous to the "°" for temperature, added small "s" for the brewtime as physical unit (as we also have weight now)

LoQue90 commented:
rebase just works, nothing to do here.
I just moved some lines, just style cleanup.
removed "and rebase" from the commit
if you want just pull: https://github.com/LoQue90/clevercoffee/tree/manuall_flush

LoQue90 commented on src/display.h:
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
+ * @brief Draw the temperature in big font at given position
+ */
+void displayTemperature(int x, int y) {
+
```
remove empty line

LoQue90 commented on src/display.h:
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
+    else {
+        u8g2.setCursor(x+69, y+14);
+    }
+    u8g2.print("s");
```
add empty line before

LoQue90 commented on src/display.h:
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
+    }
+
+    u8g2.print(brewtime / 1000, 1);
+
```
remove empty line
